github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -260,7 +260,6 @@ presubmits:
       preset-shared-images: "true"
     max_concurrency: 11
     name: pull-kubevirt-e2e-k8s-1.24-sig-storage-dv-gc
-    optional: true
```

Comment:
I would not yet make it required, it seems like the lane is not yet stable enough: https://prow.ci.kubevirt.io/job-history/gs/kubevirt-prow/pr-logs/directory/pull-kubevirt-e2e-k8s-1.24-sig-storage-dv-gc

Please do not forget that a failing required lane will block merging a PR.

Reply:
No problem, I'll battle the lane flakiness in https://github.com/kubevirt/kubevirt/pull/8351